c++/src/capnp/schema.h

```diff
@@ -341,6 +454,12 @@ class StructSchema::Field {
   //
   // If the above does not make sense, you probably don't need this method.
 
+  schema::Value::Reader getDefaultValue() const {
+    KJ_IREQUIRE((getProto().which() == schema::Field::SLOT),
+              "Must check which() before get()ing a union member.");
```

Comment:
This is a weird error message here given that we're not reading a union. Maybe: `getDefaultValueProto() cannot be called on groups`

Reply:
Updated